portfolio/src/main/webapp/script.js

```diff
@@ -29,10 +29,23 @@ function addRandomFacts() {
 
 /** Request content from server and add it to page */
 function getServerMessage() {
-    fetch('/data')
-    .then(response => response.text())
-    .then((fruits) => document.getElementById('server-message-container').innerHTML = fruits)
+    fetch('/data').then(response => response.json()).then((comments) => {
```

Comment:
easier to read if you put ".then(...)" on new lines. Example:

fetch('/data')
  .then(...)
  .then(...)

Reply:
Sounds good, thanks!